docs/source/tutorial/tutorial_template.md

```diff
@@ -34,7 +34,8 @@ The spaceflights project dependencies are stored in `src/requirements.txt`(you m
 # code quality packages
 black==22.0
 flake8>=3.7.9, <5.0
-ipython>=7.31.1, <8.0
+ipython>=7.31.1, <8.0; python_version < '3.8'
+ipython~=8.10; python_version >= '3.8'
 isort~=5.0
 nbstripout~=0.4
```

Comment:
```suggestion
nbstripout~=0.4
ruff~=0.0.290
```

Reply:
Since we're updating this page, might as well add `ruff` to reflect the recent changes to the `requirements.txt` from [kedro-starters](https://github.com/kedro-org/kedro-starters/blob/main/spaceflights/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/requirements.txt).